### PR TITLE
fix: io4 identification for some models

### PR DESCRIPTION
**Pull request by Lash-L**
Add additional IO4 model bytes

### CODE REVIEW

**Comment by Lash-L** [comment]
Will fix it all later, just wanted to get it started so it was on my github and I could remember to do it

File: src/oralb_ble/parser.py [hunk 1]
```diff
@@ -194,7 +194,7 @@ class ModelDescription:
 
 BYTES_TO_MODEL = {
     b"\x062k": Models.IOSeries67,
-    b"\x074\x0c": Models.IOSeries4,
+    b"\x074": Models.IOSeries4,
```
Comment (bdraco) [inline src/oralb_ble/parser.py:197]: These all need to be 3 bytes long since `device_bytes = data[0:3]`

**Comment by Lash-L** [comment]
@bdraco good now

**Review by bdraco — APPROVED** [review]

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -39,7 +39,7 @@ repos:
       - id: pyupgrade
         args: [--py37-plus]
   - repo: https://github.com/PyCQA/isort
-    rev: 5.10.1
+    rev: 5.12.0
```
Comment (bdraco) [inline .pre-commit-config.yaml:42]: This probably should have been a separate PR

Comment (Lash-L) [reply .pre-commit-config.yaml:42]: Can do for next time. Sounds good. Was just causing an issue with ci/cd pipeline

**Pull request merged**